master/buildbot/status/build.py

```diff
@@ -386,6 +381,9 @@ def updateBuildRequestIDs(self, brids):
     def setOwners(self, owners):
         self.owners = owners
 
+    def setUserID(self, user_id):
```

Comment:
Did you add this method only for consistency, I mean all properties have setter/getter so the new one also should have, or you use it somewhere?

Reply:
Both. I adjust setter for current style and use it in `process.Build.setupBuild`